internal/compiler/widgets/fluent-base/button.slint

```diff
@@ -7,21 +7,22 @@ import { FocusBorder } from "components.slint";
 export component Button {
     callback clicked;
 
-    in property <string> text <=> i-text.text;
+    in property <string> text;
     in property <image> icon;
     in property <bool> primary;
     in property <bool> enabled <=> i-touch-area.enabled;
     in property <bool> checkable;
     out property <bool> has-focus: i-focus-scope.has-focus;
     out property <bool> pressed: self.enabled && i-touch-area.pressed;
     in-out property <bool> checked;
+    property <brush> text-color: primary || checked ? Palette.text-on-accent-primary : Palette.text-primary;
```

Comment:
Please add `text-color` property to the start of the component and add a `private`, check `listview.slint`. Order should be 1. private properties, 2. callbacks, 3. in properties, 4. out properties, 5. in-out properties, 6. children, 7. functions, 8. states, 9. animations.

Reply:
Done